images/build/Dockerfiles/vso.bullseye.Dockerfile

```diff
@@ -0,0 +1,325 @@
+FROM mcr.microsoft.com/mirror/docker/library/buildpack-deps:bullseye AS main
+
+# Install basic build tools
+# Configure locale (required for Python)
+# NOTE: Do NOT move it from here as it could have global implications
+RUN LANG="C.UTF-8" \
+    && apt-get update \
+    && apt-get upgrade -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        git \
+        make \
+        unzip \
+        # The tools in this package are used when installing packages for Python
+        build-essential \
+        # swig3.0 \
```

Comment:
There are a couple of lines that are now commented-out in this file (lines 15, 49, 53-55, 159, 280-285, 296) -- are we able to just delete them?

Reply:
Thanks Cormac, I've moved to the PR to draft while I cleanup a little more 😃